test.js

```diff
@@ -87,3 +88,24 @@ test('end without match', async () => {
     }
   )
 })
+
+test('signal', async () => {
+  const stream = new PassThrough()
+  const controller = new AbortController()
+  const { signal } = controller
```

Comment:
Have you considered using `AbortController.timeout` instead the manual implementation?

https://nodejs.org/dist/latest-v18.x/docs/api/globals.html#static-method-abortsignaltimeoutdelay

Reply:
This makes it so much simpler <3 I forgot that we can use this already